public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -12,17 +12,27 @@ export const NoRulesSplash = () => {
 
   if (canCreateGrafanaRules || canCreateCloudRules) {
     return (
-      <EmptyListCTA
-        title="You haven't created any alert rules yet"
-        buttonIcon="bell"
-        buttonLink={'alerting/new/alerting'}
-        buttonTitle="New alert rule"
-        proTip="you can also create alert rules from existing panels and queries."
-        proTipLink="https://grafana.com/docs/"
-        proTipLinkTitle="Learn more"
-        proTipTarget="_blank"
-        onClick={() => logInfo(LogMessages.alertRuleFromScratch)}
-      />
+      <>
+        <EmptyListCTA
+          title="You haven't created any alert rules yet"
+          buttonIcon="bell"
+          buttonLink={'alerting/new/alerting'}
+          buttonTitle="New alert rule"
+          proTip="you can also create alert rules from existing panels and queries."
+          proTipLink="https://grafana.com/docs/"
+          proTipLinkTitle="Learn more"
+          proTipTarget="_blank"
+          onClick={() => logInfo(LogMessages.alertRuleFromScratch)}
+        />
+
+        <EmptyListCTA
+          title=""
+          buttonIcon="bell"
```

Comment:
Nit: I'd change or remove the `bell` icon because it might be misleading in context of recording rules